danish/velkommen.md

```diff
@@ -0,0 +1,5 @@
+Du befinder på en mark i Vestjylland tilhørende kartoffelavler Oluf Sand og dennes hustru Gertrud. Sammen med dig er Fritz, Hansi og Günther, der skal finde en nøgle til the Playdåse som spiller Gammel Noks livsmelodi (Gammel Nok bliver i sin seng).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.